<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkRadiomicsTransformationViewControls</class>
 <widget class="QWidget" name="QmitkRadiomicsTransformationViewControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>980</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="4" column="0">
    <widget class="QToolBox" name="toolBox">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>430</width>
        <height>846</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Multi-Resolution Pyramid</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Number of Levels:   </string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="m_NumberOfLevels">
             <property name="minimum">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Return result as double: </string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QCheckBox" name="m_resultAsDouble">
             <property name="text">
              <string/>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="buttonExecuteMultiresolution">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Execute Multi-Resolution Pyramid</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_4">
      <attribute name="label">
       <string>Resample Image</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <layout class="QFormLayout" name="formLayout_4">
           <item row="0" column="0">
            <widget class="QCheckBox" name="checkResampleX">
             <property name="text">
              <string>Dimension X</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="checkResampleY">
             <property name="text">
              <string>Dimension Y</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="checkResampleZ">
             <property name="text">
              <string>Dimension Z</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QDoubleSpinBox" name="doubleSpinDimensionX">
             <property name="decimals">
              <number>7</number>
             </property>
             <property name="maximum">
              <double>40000000.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QDoubleSpinBox" name="doubleSpinDimensionY">
             <property name="decimals">
              <number>7</number>
             </property>
             <property name="maximum">
              <double>40000000.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QDoubleSpinBox" name="doubleSpinDimensionZ">
             <property name="decimals">
              <number>7</number>
             </property>
             <property name="maximum">
              <double>40000000.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Interpolation mode</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Grid alignment method</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="comboInterpolationMode">
             <item>
              <property name="text">
               <string>Linear</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>B-Spline</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Nearest Neighbour</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>WSinc Hamming</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>WSinc Welch</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QComboBox" name="comboAxisAlignment">
             <item>
              <property name="text">
               <string>Origin aligned</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Center aligned</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Same size (adapts final dimension)</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QCheckBox" name="checkResamplingOutputAsDouble">
             <property name="text">
              <string>Output as double</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QCheckBox" name="checkResampleAsMask">
             <property name="text">
              <string>Resample as mask</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QCheckBox" name="checkResamplingRoundOutput">
             <property name="text">
              <string>Round output</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="buttonResampleImage">
           <property name="text">
            <string>Resample Image</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>430</width>
        <height>846</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Wavelet Transformation</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QFormLayout" name="formLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Number of Levels:  </string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="m_WaveletNumberOfLevels">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>20000</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Number of Bands: </string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="m_WaveletNumberOfBands">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>2000</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Border Condition: </string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Wavelet: </string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="m_WaveletBorderCondition">
             <item>
              <property name="text">
               <string>Constant</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Periodic</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Zero Flux Neumann</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="m_WaveletWavelet">
             <item>
              <property name="text">
               <string>Shannon</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Simoncelli</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Vow</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Held</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCommandLinkButton" name="m_WaveletExecuteButton">
         <property name="text">
          <string>Execute Wavelet Transformation</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>430</width>
        <height>846</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Laplacian of Gaussian</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="1">
            <widget class="QDoubleSpinBox" name="m_LoGSigma">
             <property name="decimals">
              <number>4</number>
             </property>
             <property name="minimum">
              <double>1.000000000000000</double>
             </property>
             <property name="maximum">
              <double>1000000000000.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Gaussian Sigma: </string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Return result as double: </string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QCheckBox" name="m_LogResultAsDouble">
             <property name="text">
              <string/>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="m_ExecuteLOG">
           <property name="text">
            <string>Execute Laplacian of Gaussian</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="m_InputImageGroup"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
